doc: align landing pages with standard pattern (#1337)
canonical/microcloud: doc/reference/index.md

> The reference material in this section provides technical information about MicroCloud.

Reviewer: ```suggestion
This section provides technical reference guides for MicroCloud.
```

> (ref-microcloud)=

Reviewer: ```suggestion
```

## MicroCloud Cluster Manager

> Learn about the MicroCloud Cluster Manager architecture and how it connects
> multiple clusters to a centralized UI tool.

Reviewer: ```suggestion
Consult these pages for information about the system architecture and API of the MicroCloud Cluster Manager, which is used for managing multiple MicroCloud clusters.
```

"Learn about" sounds like tutorial or explanation language in Diataxis terms.

```{toctree}
:maxdepth: 1
Cluster Manager architecture </reference/cluster-manager-architecture>
/reference/cluster-manager-api
```

## Commands

> Learn how to perform MicroCloud operations through the CLI.

Reviewer: ```suggestion
Use this command reference to perform common MicroCloud operations through the CLI.
```

"Learn how to" sounds like a tutorial or how-to guide.
